### PR TITLE
[v6r13] Certification fixes

### PR DESCRIPTION
**Pull request by chaen**
* Infinite loop in listing directory on SRM2 storage
* Incompatible return between LFC/DFC for listDirectory

### CODE REVIEW

File: DataManagementSystem/Client/DataManager.py [hunk 1]
```diff
@@ -190,7 +190,7 @@ def __getCatalogDirectoryContents( self, directories ):
     allFiles = {}
     while len( activeDirs ) > 0:
       currentDir = activeDirs[0]
-      res = returnSingleResult( self.fc.listDirectory( currentDir ) )
+      res = returnSingleResult( self.fc.listDirectory( currentDir, verbose = True ) )
```
Comment (phicharp) [inline DataManagementSystem/Client/DataManager.py]: I always found this "feature" insane (sorry). Using a "verbose" keyed argument for defining what the return value is is really misleading... In addition the output structure of a method should always be the same IMHO

**Comment by phicharp** [comment]
Review OK...
The "feature" of "verbose = True" should be fixed throughout the code...

**Comment by chaen** [comment]
shit !! I wanted to put it in the rel-v6r13 !!! Do you backport or should I redo it ?

**Comment by atsareg** [comment]
We are getting tired :( . Usually I check that the target branch is correct. Since this is my fault ( partly ) I will back port it.

**Comment by chaen** [comment]
I still have the branch so I can just do another PR if you want...

**Pull request merged**